learn/web/eip-1271.md

```diff
@@ -138,7 +135,7 @@ const getSafeMessageHash = async (
 };
 ```
 
-Now we can query the state of the message from the tx-service's endpoint for messages: https://safe-transaction-goerli.staging.5afe.dev/api/messages/<SafeMsgHash>
+Now we can query the state of the message from the network-specific transaction service endpoint for messages: https://safe-transaction-<Network>.safe.global/api/messages/<SafeMsgHash>
```

Comment:
```suggestion
Now we can query the state of the message from the network-specific transaction service endpoint for messages: `https://safe-transaction-<Network>.safe.global/api/v1/messages/<SafeMsgHash>`.

For example here is an example response from Goerli: `https://safe-transaction-goerli.safe.global/api/v1/messages/`.

For other network endpoints, see [Available Services](/learn/safe-core-api/available-services.md).

```

Reply:
Do you have a message hash URL we can use as an example here? Preferably Goerli but any example is fine.

Reply:
I just created an [off-chain signature](https://safe-transaction-goerli.safe.global/api/v1/messages/0x7f4032fd13f27c5fce4694a1a6b79f65c68656da4eee4347a414f5bcec915b39/) specifically for the docs and added your suggestions.